server/src/client_api.rs

```diff
@@ -134,6 +162,27 @@ fn get_feature_filter(
     Ok((validated_token, filter_set, query))
 }
 
+fn get_delta_filter(
+    edge_token: &EdgeToken,
+    token_cache: &Data<DashMap<String, EdgeToken>>,
+    filter_query: Query<FeatureFilters>,
+) -> EdgeResult<DeltaFilterSet> {
+    let validated_token = token_cache
+        .get(&edge_token.token)
+        .map(|e| e.value().clone())
+        .ok_or(EdgeError::AuthorizationDenied)?;
+
+    let query_filters = filter_query.into_inner();
+
+    let delta_filter_set = DeltaFilterSet::default().with_filter(combined_filter(
+        100,
```

Comment:
What's 100 here?

Reply:
Thanks, this was oversight actually. fixed.